testing/testing.gni

```diff
@@ -39,16 +43,31 @@ template("test_fixtures") {
     ]
   }
 
-  copy(fixtures_copy_target_name) {
-    sources = invoker.fixtures
-    outputs = [
-      "$fixtures_location/{{source_file_part}}",
+  fixtures_kernel_target_name = target_name + "_kernel"
+  prebuilt_dart_action(fixtures_kernel_target_name) {
+    script = "$root_out_dir/frontend_server.dart.snapshot"
+
+    fixture_paths = []
+    foreach(fixture, invoker.fixtures) {
+      fixture_paths += [ rebase_path(fixture) ]
+    }
+    inputs = fixture_paths
+    outputs = ["$fixtures_location/kernel_blob.bin"]
+
+    args = [
+      "--sdk-root", rebase_path("$root_out_dir/flutter_patched_sdk"),
+      "--strong",
+      "--target", "flutter",
+      "--output-dill", rebase_path("$fixtures_location/kernel_blob.bin"),
```

Comment:
I am not entirely sure but if you put these files in the inputs sections, it will re-run the target if they change on disk.

Reply:
SGTM - will update.